Conceptpower+Spring/src/main/java/edu/asu/conceptpower/rest/TypeIdLookup.java

```diff
@@ -87,6 +94,6 @@
 		HttpHeaders responseHeaders = new HttpHeaders();
         responseHeaders.add("Content-Type", "text/html; charset=utf-8");
         
-        return new ResponseEntity<String>(msg.getXML(xmlEntry), responseHeaders, HttpStatus.OK);
+        return new ResponseEntity<String>(entry, responseHeaders, HttpStatus.OK);
 	}
 }
```

Comment:
Same problem. The old code was using tab indentation. Changed everything to space indentation.